docs/fx-core/spfx-help.md

```diff
@@ -0,0 +1,72 @@
+# SPFx prerequisites troubleshoot
+
+## Overview
+
+For SPFx app, Teams Toolkit uses Yeoman Generator for scaffolding. This requires both [Yeoman CLI](https://github.com/yeoman/yo) and the correct SPFx generator version to be installed.
+
+As the default behavior, Teams Toolkit will try to install them locally under `HOME/.fx`. Should the installation fail, we would revert to use your globally installed ones.
+
+## Remediation Steps
+### Step 1: Disable Prerequisite Checker 
+
+Go to _Manage > Extension > Teams Toolkit > SPFx Prerequisite Check_
```

Comment:
User can go to settings either by Manage->Settings->Teams Toolkit or run 'Preferences: Open User Settings'